src/appleseed-max-impl/appleseedinteractive/interactivetilecallback.cpp

```diff
@@ -37,36 +41,40 @@ namespace
 }
 
 InteractiveTileCallback::InteractiveTileCallback(
-    Bitmap*                         bitmap,
-    IIRenderMgr*                    iimanager,
-    asr::IRendererController*       render_controller)
-    : TileCallback(bitmap, &m_rendered_tile_count)
-    , m_bitmap(bitmap)
-    , m_iimanager(iimanager)
-    , m_renderer_ctrl(render_controller)
+    Bitmap*                     bitmap,
+    IIRenderMgr*                iimanager,
+    asr::IRendererController*   render_controller)
+  : TileCallback(bitmap, &m_rendered_tile_count)
```

Comment:
I should have changed &m_rendered_tile_count to nullptr. Will fix it in next PR.

Reply:
Cool!